framework/decode/dx12_dump_resources.cpp

```diff
@@ -1043,9 +1043,10 @@ bool CaptureGPUAddrMatchDescriptorHeap(const D3D12_GPU_DESCRIPTOR_HANDLE capture
     auto increment            = (*heap_info.capture_increments)[heap_info.descriptor_type];
     auto capture_gpu_addr_end = heap_info.capture_gpu_addr_begin + heap_info.descriptor_count * increment;
 
-    bool is_match = true ? (heap_info.capture_gpu_addr_begin <= capture_gpu_addr.ptr &&
-                            capture_gpu_addr.ptr <= capture_gpu_addr_end)
-                         : false;
+    bool is_match =
```

Comment:
Can you just use `bool is_match = heap_info.capture_gpu_addr_begin <= capture_gpu_addr.ptr && capture_gpu_addr.ptr <= capture_gpu_addr_end;`?

(same below)

Reply:
Thank you. Fixed.